examples/chart/teleport-cluster/templates/auth/deployment.yaml

```diff
@@ -0,0 +1,265 @@
+{{- $auth := mustMergeOverwrite .Values .Values.auth -}}
+{{- $replicated := gt (int $auth.highAvailability.replicaCount) 1 -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-auth
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "teleport-cluster.auth.labels" . | nindent 4 }}
+    app: {{ .Release.Name }}
+{{- if $auth.annotations.deployment }}
+  annotations: {{- toYaml $auth.annotations.deployment | nindent 4 }}
+{{- end }}
+spec:
+  replicas: {{ $auth.highAvailability.replicaCount }}
+{{- if and $replicated $auth.highAvailability.minReadySeconds }}
+  minReadySeconds: {{ $auth.highAvailability.minReadySeconds }}
+{{- end }}
+  strategy:
+{{- if $replicated }}
```

Comment:
`type` is set in the `else` case below; does it have to be set here too or does setting `rollingUpdate` remove the need for `type` completely?

Reply:
default type is `RollingUpdate`, but it does look clearer when explicitly set on both sides of the `if`